README.md

```diff
@@ -817,6 +817,19 @@ nt-dates/data/](https://amiibereval2018.wordpress.com/important-dates/data/)
 
 <a id="German-header"></a>
 ### German
+#### RP-Mod & RP-Crowd: Moderator- and Crowd-Annotated German News Comment Datasets
+* Link to publication: [https://datasets-benchmarks-proceedings.neurips.cc/paper/2021/file/c9e1074f5b3f9fc8ea15d152add07294-Paper-round2.pdf](https://datasets-benchmarks-proceedings.neurips.cc/paper/2021/file/c9e1074f5b3f9fc8ea15d152add07294-Paper-round2.pdf)
+* Link to data: [https://zenodo.org/record/5291339#.Ybr_9VkxkUE](https://zenodo.org/record/5291339#.Ybr_9VkxkUE)
+* Task description: Binary (Offensive or Not), Multi-class/-label (sexism, racism, threats, insults, profane language, meta, advertisement).
+* Details of task: The comments originate from a large German newspaper and are annotated by professional moderators (community managers). Additionally, each comment was further annotated by five different crowd-workers.
+* Size of dataset: 85,000
+* Percentage abusive: 0.08 
```

Comment:
Is this really just 8 in 10 000 items offensive? Could this number be clarified with the addition of a percent symbol %?

Reply:
You are right. This might be confusing. I adjusted it accordingly.